docs/docs/integrations/tools/zenguard.ipynb

```diff
@@ -0,0 +1,130 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# LangChain-ZenGuard\n",
+    "\n",
+    "<a href=\"https://colab.research.google.com/github/ZenGuard-AI/langchain/blob/nuradil%2Fintegration%2Ftool%2Fzenguard/docs/docs/integrations/tools/zenguard.ipynb\" target=_parent><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"></a>\n",
```

Comment:
is this correct link? i.e. ideally it should not be from personal accounts, but better from company account.

Reply:
corrected